platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -64,16 +64,6 @@ typedef NS_ENUM(NSUInteger, MGLCircleTranslationAnchor) {
  ### Example
 
  ```swift
- let layer = MGLCircleStyleLayer(identifier: "circles", source: population)
```

Comment:
Thanks for fixing this. In the future, consider running `make style-code` afterwards to see if it would add anything that isn’t already in the code base.